### PR TITLE
LC #380 rename NEXT_PIPE_OFFSET to PIPE_OFFSET and add offsets map in Node

### PR DESCRIPTION
**Pull request by LorenzoCianciaruso**

### CODE REVIEW

File: registry-core/src/main/java/com/tesco/aqueduct/registry/model/Node.java [hunk 1]
```diff
@@ -30,11 +30,6 @@
      */
     private final long offset;
 
-    /**
-     * The pipe state of this node
-     */
-    private final PipeState pipeState;
-
```
Comment (ashish-sharma09) [inline registry-core/src/main/java/com/tesco/aqueduct/registry/model/Node.java]: Why have we removed Pipe State?

Comment (ashish-sharma09) [reply registry-core/src/main/java/com/tesco/aqueduct/registry/model/Node.java]: I think it is still an useful indicator

Comment (LorenzoCianciaruso) [reply registry-core/src/main/java/com/tesco/aqueduct/registry/model/Node.java]: yeah sorry my bad, that wasn't intended

Comment (LorenzoCianciaruso) [reply registry-core/src/main/java/com/tesco/aqueduct/registry/model/Node.java]: restored

**Pull request merged**